lib/js-file.js

```diff
@@ -78,22 +78,44 @@ JsFile.prototype = {
         this._disabledRuleIndex = [];
 
         var comments = this.getComments();
-        var commentRe = /(jscs\s*:\s*(en|dis)able)(.*)/;
+        var blockRe = /(jscs\s*:\s*(en|dis)able)(.*)/;
+        var lineRe = /(jscs\s*:\s*ignore)(\s+\w.*)/;
```

Comment:
In some guides it is preferable to move them in special variables with prefixes, it seems author followed that guide.

> Would you prefer them not to be variables?

It is fine with me, just a comment would suffice